addons/web_editor/static/src/components/media_dialog/media_dialog.xml

```diff
@@ -4,21 +4,7 @@
     <Dialog contentClass="contentClass"
         size="size"
         title="title">
-        <div class="o_notebook">
-            <div class="o_notebook_headers">
-                <ul class="nav nav-tabs" role="tablist">
-                    <li t-foreach="tabs" t-as="tab" t-key="tab.id" class="nav-item cursor-pointer">
-                        <a class="nav-link" t-att-class="{ active:  tab.id === state.activeTab }" t-on-click="() => state.activeTab = tab.id"><t t-esc="tab.title"/></a>
-                    </li>
-                </ul>
-            </div>
-            <!-- Tab panes -->
-            <div class="o_notebook_content tab-content">
-                <t t-foreach="tabs" t-as="tab" t-key="tab.id">
-                    <div class="tab-pane fade" t-att-class="tab.id === state.activeTab ? 'active show' : ''"><t t-component="tab.Component" t-props="tab.props"/></div>
-                </t>
-            </div>
-        </div>
+        <Notebook pages="tabs" onPageUpdate.bind="onTabChange" defaultPage="state.activeTab"/>
```

Comment:
Funny enough, it solves an issue I was fixing differently here https://github.com/odoo/odoo/pull/100039/commits/6b977f90f2025fd28e5aced9596f3164810ca232 (in the end, it's the same fix, because only the active page is rendered).
Other than that there are a few changes in the style (`bg-white pt-3` is added on the notebook content, so the icons are black on the icons tab, and on mobile it does not behave the same way), maybe @qsm-odoo can have a look at it ?

Reply:
I tested again and my bad, it's the same behaviour in mobile, sorry for that: I tested with the dev tools, resizing to get to mobile dimensions but that never happens. On mobile, starting the media dialog is the same with the Notebook component or without !